src/ensembl/src/content/app/browser/chromosome-navigator/ChromosomeNavigator.tsx

```diff
@@ -1,12 +1,25 @@
 // TODO: think about the name; perhaps something like MiniMap is a better option?
```

Comment:
Please check if we need to retain this comment.

Reply:
That depends on whether we are ok with calling this thing ChromosomeNavigator, or whether we would prefer a name that is less tied to the concept of chromosome (since the nucleic acid strands displayed in genome browser can also be other than chromosomes).

Reply:
I think keeping it as `ChromosomeNavigator` is fine as long as we can understand it. Any thought's @imran-ebi ?

Reply:
I don't have a problem with the name. We can leave it as it is if we don't have a better name for it.

Reply:
✅